portality/formcontext/forms.py

```diff
@@ -674,8 +674,8 @@ class ArticleForm(Form):
 
     volume = IntegerField("Volume", [validators.Optional()])
     number = IntegerField("Issue", [validators.Optional()])
-    start = IntegerField("Start", [validators.Optional()])
-    end = IntegerField("End", [validators.Optional()])
+    start = StringField("Start", [validators.Optional()])
```

Comment:
Good find. Is there an issue open for this? volume and issue are also stored as unicode in the data structure, so they should probably also have the same treatment.

Reply:
https://github.com/DOAJ/doajPM/issues/2654

Reply:
This is how it's shown on the commit: 

<img width="329" alt="Screenshot 2020-12-03 at 17 02 46" src="https://user-images.githubusercontent.com/6668406/101062282-5e129580-3589-11eb-9328-cac3c8ea29dd.png">

Reply:
great, if you use 'Fixes' rather than 'See' Github will know to close the issue in question on merge https://guides.github.com/features/issues/#notifications

Reply:
Yeah, I just don’t use it here because I figured that Dom might want to check / test them before closing them!

Reply:
good point, we'll keep the issues open and not use `fixes`. (although Dom had already closed these prematurely as soon as you'd issued the PR!)